docs/lib/QUICKSTART.md

```diff
@@ -286,7 +288,7 @@ A few things to note in the code above:
 
 - If code runs in the context of `private_button`, then any object instantiated inside that code belongs to the `private_button` heap and stack. This means that in the example above, the `InterruptIn` object is private to the `private_button` box. The same applies to the dynamically allocated buffer `uvisor_ctx->buffer`.
 - You can access the content of the private memory `PrivateButtonStaticMemory` using the `void * const __uvisor_ctx` pointer, which uVisor maintains. You need to cast this pointer to your own context type. In this example we used a pre-processor symbol to improve readability.
-- The `InterruptIn` object triggers the registration of an interrupt slot. Because that code runs in the context of the `private_button` box, the push-button IRQ belongs to that box. If you want to use the IRQ APIs directly, read the [NVIC APIs section](#the-nvic-apis) below.
+- The `InterruptIn` object triggers the registration of an interrupt slot using the NVIC APIs under the hood. If you want to use the IRQ APIs directly, read the [NVIC APIs section](#the-nvic-apis) below. The push-button IRQ has been registered to the `private_button` box via an IRQ ACL, and hence can only be used from this box. If you configured `InterruptIn` from the public box, you would get a uVisor fault.
```

Comment:
- In the first sentence, "under the hood" is an idiom, which can be confusing for our non-native-English readers. Please replace it with more precise language.
- In the second sentence, "has been registered" is passive voice. Would it keep the same meaning if we the start of the sentence to "An IRQ ACL has registered the push-button IRQ to the `private_button` box..."?
- Query: In the second sentence, would the meaning stay the same if we changed "and hence can only be used from this box." to "so you can only use the push-button IRQ from this box." for active voice?
- For consistent mood, please change the last sentence to "If you configure `InterruptIn` from the public box, you get a uVisor fault." or "Configuring `InterruptIn` from the public box causes a uVisor fault.